### ISSUE TITLE
JAWS - Autocomplete does not announce list opening when re-focusing autocomplete with selection

### ISSUE THREAD

**Issue by DaveBest99** [angular/components#3376]
#### Bug, feature request, or proposal:
Bug: Keyboard user cannot open list box with alt+down_arrow

#### What is the expected behavior?
Press Enter to select the highlighted choice and close the drop-down list. This mimics the behavior of the HTML select element. Moving focus away from the Edit Combo box (pressing tab or shift+tab), and then moving focus back to the Edit Combo box should display the selected list item and pressing alt+down_arrow opens the list box allowing the user to press down arrow into the list.

#### What is the current behavior?
Currently the keyboard user cannot open the drop-down list without deleting the typed text in the Edit box first.

#### What is the use-case or motivation for changing an existing behavior?
To create a good keyboard user experience.

#### Is there anything else we should know?
Moving focus back to the Autocomplete Edit Combo box displays the selected list item, but the user is not able to open the list to find another list selection. Pressing alt+down_arrow does not open the list box, and pressing Enter key simply reselects the current list item. The user can delete or type characters to move the focus through the list, but it is not obvious that the focus is on the Edit box, as the screen reader does not identify the element to be editable.

**Comment by kara** [comment]
Related: Related: https://github.com/angular/material2/issues/3416
